File: logica/Boleta.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package logica;

/**
 *
 * @author Daniela Huertas
 */
public class Boleta {
    private int idBoleta;
    private int idAsistente;
    private int idFactura;
    private String pulep;
    private int precio;

    // Constructor
    public Boleta(int idBoleta, int idAsistente, int idFactura, String pulep, int precio) {
        this.idBoleta = idBoleta;
        this.idAsistente = idAsistente;
        this.idFactura = idFactura;
        this.pulep = pulep;
        this.precio = precio;
    }

    // Getters y Setters
    public int getIdBoleta() {
        return idBoleta;
    }

    public void setIdBoleta(int idBoleta) {
        this.idBoleta = idBoleta;
    }

    public int getIdAsistente() {
        return idAsistente;
    }

    public void setIdAsistente(int idAsistente) {
        this.idAsistente = idAsistente;
    }

    public int getIdFactura() {
        return idFactura;
    }

    public void setIdFactura(int idFactura) {
        this.idFactura = idFactura;
    }

    public String getPulep() {
        return pulep;
    }

    public void setPulep(String pulep) {
        this.pulep = pulep;
    }

    public int getPrecio() {
        return precio;
    }

    public void setPrecio(int precio) {
        this.precio = precio;
    }
}

File: logica/Boleta.php
<?php
class Boleta {
    private $idBoleta, $idAsistente, $idFactura;
    
    // Getters y Setters
    public function getIdBoleta() {
        return $this->idBoleta;
    }

    public function getIdAsistente() {
        return $this->idAsistente;
    }

    public function getIdFactura() {
        return $this->idFactura;
    }

    public function getPrecio() {
        return $this->precio;
    }

    public function setIdBoleta($idBoleta) {
        $this->idBoleta = $idBoleta;
    }

    public function setIdAsistente($idAsistente) {
        $this->idAsistente = $idAsistente;
    }

    public function setIdFactura($idFactura) {
        $this->idFactura = $idFactura;
    }

    // Constructor
    public function __construct($idBoleta=null, $idAsistente=null, $idFactura=null) {
        $this->idBoleta = $idBoleta;
        $this->idAsistente = $idAsistente;
        $this->idFactura = $idFactura;
    }
}

